docker/scripts/mongo-init.sh

```diff
@@ -8,25 +8,32 @@ until mongosh --eval "rs.status()" | grep "stateStr: 'PRIMARY'"; do
 done
 echo "Mongo started"
 
-# import event provider db (takes around 10-15 mins)
+cd /dumps
 
 if [ ! -f eventProviderV2 ]; then
     echo "eventProviderV2 not found skipping..."
 else
     echo "eventProviderV2 found, importing"
-    mongorestore --uri="mongodb://localhost:27017" --gzip --archive=eventProviderV2 --nsInclude=eventProviderV2.* -v
+    mongorestore --uri="mongodb://localhost:27017" --gzip --archive=eventProviderV2 --nsInclude=eventProviderV2.* -v && rm eventProviderV2
+fi
+
+if [ ! -f eventProviderV2Pruned ]; then
+    echo "eventProviderV2Pruned not found skipping..."
+else
+    echo "eventProviderV2Pruned found, importing"
+    mongorestore --uri="mongodb://localhost:27017" --gzip --archive=eventProviderV2Pruned --nsFrom=eventProviderV2Pruned.* --nsTo=eventProviderV2.* -v && rm eventProviderV2Pruned
 fi
 
 if [ ! -f exchangeV2 ]; then
     echo "exchangeV2 not found skipping..."
 else
     echo "exchangeV2 found, importing"
-    mongorestore --uri="mongodb://localhost:27017" --gzip --archive=exchangeV2 --nsInclude=exchangeV2.* -v
+    mongorestore --uri="mongodb://localhost:27017" --gzip --archive=exchangeV2 --nsInclude=exchangeV2.* -v && rm exchangeV2
```

Comment:
exchange also can be pruned, IMO 1 month old data is more than enough for MM and most use cases.